src/screens/CreatePost/index.js

```diff
@@ -449,11 +450,13 @@ const CreatePost = () => {
   };
 
   const postV2 = async () => {
+    setLoadingPost(true);
     if (!isEmptyMessageAllowed()) {
       showMessage({
         message: StringConstant.createPostFailedNoMessage,
         type: 'danger'
       });
+      setLoadingPost(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Setting `loadingPost` to true at the start of the `postV2` function is correct. However, it's important to ensure that `setLoadingPost(false)` is called in all possible exit points of the function to avoid leaving the application in a loading state indefinitely. In this case, you've correctly set `loadingPost` to false when the message is empty.
```diff
+ setLoadingPost(true);
+ if (!isEmptyMessageAllowed()) {
+   showMessage({
+     message: StringConstant.createPostFailedNoMessage,
+     type: 'danger'
+   });
+   setLoadingPost(false);
+   return true;
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->